x-pack/plugins/profiling/public/components/no_data_page.tsx

```diff
@@ -265,43 +268,71 @@ docker.elastic.co/observability/profiling-agent:${hostAgentVersion} /root/pf-hos
+            </EuiCodeBlock>
           ),
         },
         {
           title: i18n.translate('xpack.profiling.tabs.symbols.step2', {
-            defaultMessage: 'Generate an Elasticsearch token:',
+            defaultMessage: 'Generate an Elasticsearch token',
           }),
           content: (
-            <EuiLink
-              target="_blank"
-              href={`https://www.elastic.co/guide/en/kibana/master/api-keys.html`}
-            >
-              {i18n.translate('xpack.profiling.tabs.symbols.step2.instructions', {
-                defaultMessage: 'Instructions here',
-              })}
-            </EuiLink>
+            <EuiText>
+              <EuiLink
+                target="_blank"
+                href={`https://www.elastic.co/guide/en/kibana/master/api-keys.html`}
+              >
+                {i18n.translate('xpack.profiling.tabs.symbols.step2.instructions', {
+                  defaultMessage: 'Instructions here',
+                })}
+              </EuiLink>
+            </EuiText>
           ),
         },
         {
           title: i18n.translate('xpack.profiling.tabs.symbols.step3', {
-            defaultMessage:
-              'Fetch the symbols endpoint from the Cloud console, navigating in the Integrations Server section',
-          }),
-        },
-        {
-          title: i18n.translate('xpack.profiling.tabs.symbols.step4', {
-            defaultMessage: 'Run the following command, replacing the placeholders between <>:',
+            defaultMessage: 'Upload symbols',
           }),
           content: (
-            <EuiCodeBlock paddingSize="s" isCopyable>
-              {`./symbtool push-symbols executable --url <symbols endpoint> --api-key <api key> --help`}
-            </EuiCodeBlock>
+            <div>
+              <EuiCodeBlock paddingSize="s" isCopyable>
+                {`./symbtool push-symbols executable -u "${symbolUrl}" -t <ES token> -e <my executable>`}
+              </EuiCodeBlock>
+              <EuiSpacer size="m" />
+              <EuiText>
+                <FormattedMessage
+                  id="xpack.profiling.tabs.symbols.step3.replace"
+                  defaultMessage="Replace {es_token} etc. with the actual values. You can pass {help} to obtain a list of other arguments."
+                  values={{
+                    es_token: <EuiCode>{`<ES token>`}</EuiCode>,
+                    help: <EuiCode>--help</EuiCode>,
+                  }}
+                />
+              </EuiText>
+              <EuiSpacer size="s" />
+              <EuiText>
+                <FormattedMessage
+                  id="xpack.profiling.tabs.symbols.step3.doc-ref"
+                  defaultMessage="Documentation for more advanced uses-cases is available in {link}."
```

Comment:
```suggestion
                  defaultMessage="Documentation for more advanced use cases is available in {link}."
```

Minor spelling update